recipe/prepare.bash

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash -eu
+
+env | sort
+RECIPE_DIR=${1-$RECIPE_DIR}
+SRC_DIR=${2-$SRC_DIR}
+
+VERSION=$(cd $SRC_DIR && python setup.py --version)
```

Comment:
Thanks for raising the version point on AppVeyor, @pkgw , was not aware of that issue. For whatever reason, this seemed to not affect Linux and macOS, but in retrospect it probably should.

It seems that before `python setup.py --version` was not being run in the source directory, but instead the path to `setup.py` was being provided. From past experience, have found this to be a bad idea. So have updated this line so that we switch to the source directory. As can be seen by the referenced build, the problem goes away with this change.

xref: https://ci.appveyor.com/project/jakirkham/ruamel-yaml-feedstock/build/1.0.1/job/oetlp37xtcp8eima#L586